### PR TITLE
Resolved TODO in crate-exporter.py

### PR DESCRIPTION
**Pull request by Necravisaketi**
## Proposed changes

Resolved TODO in crate-exporter.py #702 

## Types of changes

What types of changes does your code introduce to the project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Maintenance (update of libraries or other dependences)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING][contrib] doc
- [x] I have signed the [CLA][cla]
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have run **all** the existing tests locally (not just those related to my feature) and there are no errors
- [x] After the last push to the PR branch, I have run the lint script locally and there are no changes to the code base
- [x] I have updated the [RELEASE NOTES][release]
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

[cla]: https://raw.githubusercontent.com/orchestracities/ngsi-timeseries-api/master/individual_cla.pdf
    "Martel Open Source Software Individual Contributor License Agreement"
[contrib]: https://github.com/orchestracities/ngsi-timeseries-api/blob/master/CONTRIBUTING.md
    "Contributing to QuantumLeap"
[release]: https://github.com/orchestracities/ngsi-timeseries-api/blob/master/RELEASE_NOTES.md
    "QuantumLeap Release Notes"

### CODE REVIEW

File: timescale-container/crate-exporter.py [hunk 1]
```diff
@@ -69,21 +69,13 @@ class CrateSql:
 
     @staticmethod
     def to_string(x) -> str:
-        """
-        Convert the input into a Crate string.
-        E.g. input   ~~~> 'input'
-        """
```
Comment (c0c0n3) [inline timescale-container/crate-exporter.py:75]: why delete the docs?

File: timescale-container/crate-exporter.py [hunk 2]
```diff
@@ -69,21 +69,13 @@ class CrateSql:
...
-        return f"'{v}'"
+
+        return f"'{x}'"
 
     @staticmethod
     def to_quoted_identifier(x) -> str:
-        """
-        Convert the input into a Crate quoted identifier.
-        E.g. input   ~~~> "input"
-        """
```
Comment (c0c0n3) [inline timescale-container/crate-exporter.py:84]: why delete the docs?

File: timescale-container/crate-exporter.py [hunk 3]
```diff
@@ -69,21 +69,13 @@ class CrateSql:
...
-        v = str(x)  # TODO consider escaping?
-        return f'"{v}"'
+
+        return f'"{x}"'
```
Comment (c0c0n3) [inline timescale-container/crate-exporter.py:78]: how is this going to help? :-)

```python
>>> x = 'howzit"!"'
>>> f'"{x}"'
'"howzit"!""'
```

whereas the double quotes in `x` should've been escaped: `"howzit\"!\""`. Or something similar, can't remember offhand what the escape sequence is for Crate...

File: timescale-container/crate-exporter.py [hunk 4]
```diff
@@ -69,21 +69,13 @@ class CrateSql:
...
     @staticmethod
     def to_string(x) -> str:
-        """
-        Convert the input into a Crate string.
-        E.g. input   ~~~> 'input'
-        """
-        v = str(x)  # TODO consider escaping?
-        return f"'{v}'"
+
+        return f"'{x}'"
```
Comment (c0c0n3) [inline timescale-container/crate-exporter.py:73]: how is this going to help? :-)

```python
>>> x = "howzit'!'"
>>> f"'{x}'"
"'howzit'!''"
```

whereas the single quotes in `x` should've been escaped: `'howzit\'!\''`. Or something similar, can't remember offhand what the escape sequence is for Crate...

**Review by c0c0n3 — COMMENTED** [review]
@Necravisaketi nice try, but I reckon you've whipped it together a bit too quickly? :-))

**Comment by pooja1pathak** [comment]
Covered in https://github.com/orchestracities/ngsi-timeseries-api/pull/759

**Pull request closed without merging**